server/events/passwordResetInit.js

```diff
@@ -1,18 +1,33 @@
+import { readFileSync } from 'fs';
+import path from 'path';
 import flow from 'lodash/fp/flow';
 import template from 'lodash/template';
 import memoize from 'lodash/memoize';
 import juice from 'juice';
+import config from '../../yeep.config';
 
-let compileTemplate = memoize(
+const baseUrl = config.baseUrl;
+const defaultViewPath = path.resolve(__dirname, '../views/');
+const templatePath =
+  config.mail && config.mail.templatePath
+    ? path.resolve(config.mail.templatePath)
+    : defaultViewPath;
+const emailTemplateContent = readFileSync(`${templatePath}/passwordResetInit.html`, 'utf8');
+const emailTemplate = memoize(
```

Comment:
Ah, but you don't need `memoize` anymore. 

Let first create a `compileEmailTemplate.js` util with the following contents...

```javascript
import flow from 'lodash/fp/flow';
import template from 'lodash/template';

const compileEmailTemplate = flow(
    juice,
    template
  );

export default compileEmailTemplate;
```

and then change this to...

```javascript
const emailTemplate = compileEmailTemplate(readFileSync(`${templatePath}/passwordResetInit.html`, 'utf8'));
```